lib/users/routes.js

```diff
@@ -30,18 +35,203 @@ const all = (req, res, next) => {
 
 /**
  * Add a new User
+ * @param {object} req - HTTP Request
+ * @param {object} res - HTTP Response
+ * @param {object} next - Express callback
+ *
+ * @returns {promise} content-types promise
  */
-const add = () => {};
+const add = (req, res, next) => {
+  const errors = _.get(req.session, 'form.users.save.errors', {});
+  const values = _.get(req.session, 'form.users.save.content', {});
+  const merged = req.app.get('users-model') || {};
+  const cfg = _.cloneDeep(config.users);
+  delete cfg.actions.delete;
+
+  _.unset(req.session, 'form.users.save');
+
+  return types.only('users', values, [merged], config).then(only => {
+    return types.form(only, errors, config).then(form => {
+      res.render('users/add', {
+        form,
+        action: `/${config.users.base}/${config.users.actions.save}`,
+        config: cfg,
+        structure: model.structure,
+        button: config.users.actions.save,
+      });
+    });
+  }).catch(e => {
+    const err = {
+      message: e.message,
+      safe: `/${config.users.base}`,
+      status: 500,
+    };
+
+    return next(err);
+  });
+};
 
 /**
  * View/edit a User
+ * @param {object} req - HTTP Request
+ * @param {object} res - HTTP Response
+ * @param {object} next - Express callback
+ *
+ * @returns {promise} resolves/rejects promise
  */
-const one = () => {};
+const one = (req, res, next) => {
+  const id = _.get(req.session, 'form.users.edit.id', null);
+  const errors = _.get(req.session, 'form.users.save.errors', {});
+  const values = _.get(req.session, 'form.users.save.content', {});
+  const merged = req.app.get('users-model') || {};
```

Comment:
The description for password on a user edit form should explain how entering a new password will change the existing one